geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -101,6 +110,18 @@ app.MymapsDirectiveController = function($scope, $compile, gettext) {
    * @export
    */
   this.newDescription = '';
+
+  /**
+   * @type {Array.<ol.Feature>}
+   * @export
+   */
+  this.featuresList = this['features'].getArray();
```

Comment:
You should declare `this.features` with an `@export`. In this way you won't need to use the square bracket notation.